test/suites/080-dataview.js

```diff
@@ -2,12 +2,16 @@ describe('Dataview', function () {
     this.bail(true)
     this.timeout(20 * 1000)
 
-    it('Switch to "Grids" tab', function (done) {
+    it('Switch to "Dataview" tab', function (done) {
         eTT().tab('Dataview').click(done)
     })
 
-    it('Click on second "customDataviewReference" grid row', function (done) {
+    it('Click on second "customDataviewReference" dataview item', function (done) {
         eTT().dataview('customDataviewReference').select(1, done)
     })
+
+    it('Click on first "customDataviewReference1" dataview item with itemCls', function (done) {
+        eTT().dataview('customDataviewReference1').select(0, done, 'name-item');
```

Comment:
yes....working in late evenings and it may happen to escape or miss something, so good to have other eyes here :)